r2/r2/controllers/api.py

```diff
@@ -4100,14 +4100,21 @@ def POST_selectflair(self, form, jquery, user, link, flair_template_id,
 
             # XXX: gross UI code
             # Push some client-side updates back to the browser.
+
+            # remove "linkflair" and "linkflair-css_class" classes if they exist
+            jquery('.id-%s' % link._fullname).remove_link_flair_class()
```

Comment:
nit: can you rename the `remove_link_flair_class` function to `removeLinkFlairClass`?